Makefile

```diff
@@ -165,6 +165,8 @@ revendor:
 	@GO111MODULE=on go mod vendor
 	@GO111MODULE=on go mod tidy
 	@curl -sSLo hack/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/$(CR_VERSION)/hack/setup-envtest.sh
+# TODO: Remove this once k8s.io/apimachinery is upgraded to v0.20.7, see https://github.com/kubernetes/kubernetes/pull/101326
```

Comment:
It is strange that you apply such workaround for `resource.Quantity` type, but you didn't update the k/k dependencies to adopt the fix for all other types that we use. For example our types reference `corev1.ObjectReference`, `corev1.ObjectReference`, `corev1.Taint` and many others.
I suggest to update to k/k@v0.20.6 to adopt also the changes for the k/k types.